pkg/flowaggregator/flowaggregator.go

```diff
@@ -458,6 +578,70 @@ func (fa *flowAggregator) flowRecordExpiryCheck(stopCh <-chan struct{}) {
 			klog.V(4).InfoS("Total number of records exported", "count", fa.numRecordsExported)
 			klog.V(4).InfoS("Total number of flows stored in Flow Aggregator", "count", fa.aggregationProcess.GetNumFlows())
 			klog.V(4).InfoS("Number of exporters connected with Flow Aggregator", "count", fa.collectingProcess.GetNumConnToCollector())
+		case msg := <-fa.updateCh:
```

Comment:
It feels like the name fo this function (currently `flowRecordExpiryCheck`) should be changed to something more generic now.

Reply:
Like exportingCheckAndUpdate?

Reply:
Updated. Thanks/